tensorflow_addons/layers/normalizations_test.py

```diff
@@ -24,6 +24,8 @@
 from tensorflow_addons.utils import test_utils
 
 
+@test_utils.run_all_in_graph_and_eager_modes
+@tf.function
```

Comment:
What is the logic on decorating a TestCase class with `tf.function`? Does it make all eager methods run as a `tf.function`? I wasn't aware of this being used this way?

Reply:
Thanks @seanpmorgan. My original thought with `tf.function` is for [test here](https://github.com/tensorflow/addons/blob/1374b46f2d12e993fed903485b7cf8cd3e9b897e/tensorflow_addons/layers/normalizations_test.py#L56). If there is a better way to do, I'd like to have your suggestions. Thanks again.

Reply:
`tf.function` works only on python functions, it doesn't work on classes. [Code here](https://github.com/tensorflow/tensorflow/blob/befd063e093b306084642872c1c7e95317e0fee4/tensorflow/python/eager/def_function.py#L971).

Further, we'd never want to wrap any test function in `tf.function` since that'll force it to execute in a non-eager (graph) way whereas we want to test both eager and graph execution. Wrapping in one of the decorators in`test_utils` should be sufficient.

Reply:
Thanks @Squadrick.

Reply:
No no, you can wrap the class in `test_utils.run_all_in_graph_and_eager_modes`. Just delete `tf.function`.

Reply:
Thanks @Squadrick. Yea, in this case, I may need to change the test a bit, since errors will be raised by the [line](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/normalizations_test.py#L53)

Reply:
Thanks @Squadrick. I've remove `tf.function` and fix the errors cause by running `graph` mode. Can you help review when you are convenient?